float32 heading # deg
float32 yawRate # rad/s
float32 forwardVel
float32 swayVel

int32 controller_onOff

float32 headingDemand
float32 heading_error
float32 yawRateDemand

float32 s
float32 s_dot
float32 N_eq
float32 N_sw

int32 u_R
int32 u_th

float32 sw_int
